controllers/apply/under10k/fields/project-dates.js

```diff
@@ -193,29 +194,38 @@ module.exports = {
             const maxDate = moment().add(getMaxDurationMonths(), 'months');
 
             /**
-             * When projectStartDateCheck is asap
-             * we don't show the project start date question
-             * so we allow the end date to be any future date
+             * For projects in England when asap we skip the project dates
```

Comment:
Can we change `when asap` to be `when projectStartDateCheck is asap` (like before) so it's clearer what this means? Same for line 200 below.